test/fake-intake/fakeintake/server.go

```diff
@@ -38,6 +38,7 @@ func NewFakeIntake() *FakeIntake {
 
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", fi.postPayload)
```

Comment:
This PR may not be required if you handle `https://github.com/DataDog/datadog-agent/pull/15294#discussion_r1094587506` (By default you can return `http.StatusOK`).

Reply:
Using this PR to change the default behavior of the take all intake